tests/test_basic_ops_dict.py

```diff
@@ -205,6 +204,26 @@ def test_dict_get_with_default(
     assert c.get(key, default_val) == default_val
 
 
+@pytest.mark.parametrize(
+    "d,select,key,exc",
+    [
+        ({"hello": "${foo}"}, "", "hello", InterpolationKeyError),
+        (
+            {"hello": "${foo}", "foo": "???"},
+            "",
+            "hello",
+            InterpolationToMissingValueError,
+        ),
+        ({"hello": DictConfig(content="${foo}")}, "", "hello", InterpolationKeyError),
+    ],
+)
+def test_dict_get_with_default_errors(d: Any, select: Any, key: Any, exc: type) -> None:
```

Comment:
1. Use a hard coded key and access with `cfg.hello`.
2. Those are all raising. pass in the pytest.raises() object?

Reply:
1. Sure (I can remove `select` as well, I had done it this way to be consistent with the test just above). Done in 56ac731 (almost: I'm still using `c.get()` because I want to test with a default value)
2. I'm actually not passing the `pytest.raises()` object precisely because they are all raising, so it makes the definition of each parameter shorter. Do you still prefer me to change it?